src/components/SearchResults.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import algoliaSearch from 'algoliasearch';
+import useSearchResults from './useSearchResults';
+import { SEARCH_PARAM_RESOURCE } from '../constants';
+
+const renderResults = (results: algoliaSearch.Response) => {
+  const { hits } = results;
+  if (hits && hits.length) {
+    const listItems = hits.map(({ charityname, objectID }) => {
+      if (charityname && objectID) {
```

Comment:
is this check really needed? Aren't these always defined in `TResource`?

Reply:
meh, maybe and maybe not. Although this is actually what comes back from algolia, which isn't an entire `TResource`. I more just wanted to specify that something should only happen if we have these required items. I'm good either way though if you think it should be removed.

Reply:
I'll go ahead and merge for now, but this will most definitely be revisited when we hammer out how each individual listing appears.